library/markdown.php

```diff
@@ -15,7 +15,9 @@ function Markdown($text) {
 	$stamp1 = microtime(true);
 
 	# Read file and pass content through the Markdown parser
```

Comment:
For ADN it doesn't matter since this network is dead.

But the timestamp indeed is important. This had to move as well.